min_heap_binary_tree.rb

```diff
@@ -15,6 +15,26 @@ def insert(node)
     set_min_heap_state
   end
 
+  def delete(node)
+    # byebug
```

Comment:
We're all guilty of it, but try to remember to remove any `byebug`'s before you commit code to GitHub.  There is another below.